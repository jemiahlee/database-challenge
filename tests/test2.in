set var 20
get var
numequalto 20
unset var
unset var
numequalto 20
get var
END
