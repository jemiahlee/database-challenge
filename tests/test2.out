20
1
0
NULL
